vocab_files/attribute_concepts/voucher-condition.ttl

```diff
@@ -6,11 +7,12 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/nrm/010ee3c8-bfbe-47c5-adc8-068ca664173d>
     a skos:Concept ;
+    dcterms:source "O’Neill S, McCallum K, Bignall J, Laws M, and Sparrow B, (unpublished draft) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "The condition of the voucher. E.g. is 'Fresh'." ;
+    skos:definition "The condition of the voucher specimen/s collected during an ecological survey. E.g. is 'Fresh'." ;
```

Comment:
are there any predefined values?

Reply:
yes, categorical values for `voucher condition` is in this collection https://linked.data.gov.au/def/nrm/f0647b85-a014-4299-b23f-6aeb25e834bc